### ISSUE TITLE
RSEM module failed 

### ISSUE THREAD

**Issue by owenwilkins** [ewels/MultiQC#1040]
**Description of bug:**
The 'rsem' MultiQC module broke... 

**MultiQC Error log:**

```python
Module rsem raised an exception: Traceback (most recent call last):
  File "/dartfs-hpc/rc/home/d/d41294d/.conda/envs/seqqc/bin/multiqc", line 440, in multiqc
    output = mod()
  File "/dartfs-hpc/rc/home/d/d41294d/.conda/envs/seqqc/lib/python3.7/site-packages/multiqc/modules/rsem/rsem.py", line 34, in __init__
    self.parse_rsem_report(f)
  File "/dartfs-hpc/rc/home/d/d41294d/.conda/envs/seqqc/lib/python3.7/site-packages/multiqc/modules/rsem/rsem.py", line 76, in parse_rsem_report
    data['alignable_percent'] = (float(s[1]) / float(s[3])) * 100.0
ZeroDivisionError: float division by zero
```

**MultiQC run details (please complete the following):**
 - Command used to:  multiqc .
 - MultiQC Version: MultiQC v1.7
 - Method of MultiQC installation: conda

**Comment by ewels** [comment]
Thanks @owenwilkins - could you please upload a log file that triggers this error? Although I think I can guess what's going on here (you have zero reads in your dataset), it's good to have an example file to work with to be sure that the bug is fixed.

If GitHub doesn't allow the file type, just zip it or rename it to `.txt`

Phil

**Comment by owenwilkins** [comment]
Sorry, little confused by what you mean by a log file in this context.. I don’t get any multiqc files generated..

**Comment by ewels** [comment]
I mean the rsem log files that you're supplying to MultiQC.. Just above where you pasted the MultiQC log it should tell you what the last file was that MultiQC was attempting to parse.

**Comment by owenwilkins** [comment]
Ah OK. Yes you were right, this is because there are zero reads in this file due to a previous error of mine. Still want me to send it? Either way you resolved my issue!

**Comment by ewels** [comment]
Great! Would still be nice to catch the error and still display the results in the report. After all, `0` is technically a valid number of reads..! If you could send the file that would be great. I've fixed things like this before without a test file only to find that once bug 1 is fixed, others turn up later in the code.

Phil

**Comment by owenwilkins** [comment]
No problem, is emailing to your easiest?

**Comment by ewels** [comment]
Uploading to this issue is easiest if possible - just drag and drop the file onto the GitHub issue comment interface. You may need to zip or rename the file extension. Alternatively, you can make a pull-request with the file to https://github.com/ewels/MultiQC_TestData/tree/master/data/modules/rsem (this is where I want it to end up, for automated CI tests).

Phil

**Comment by ewels** [comment]
Ping 😉

**Comment by owenwilkins** [comment]
Apologies, was out last week, will look for these now

**Comment by ewels** [comment]
Hi @owenwilkins,

Any joy with finding these example files?

Phil

**Comment by owenwilkins** [comment]
Hi Phil, Apologies, I think I sent a separate email, but I left these files in a directory on our HPC that got purged before I pulled the files down, so apologies for that

**Comment by ewels** [comment]
No worries - I'll try to guess at what it looked like based on the [example RSEM data](https://github.com/ewels/MultiQC_TestData/blob/master/data/modules/rsem/sample1.cnt) we already have, and see if I can replicate the error traceback above.

**Comment by ewels** [comment]
Fixed by @ErikDanielsson in #1474 (complete with synthesised example data) - this will be included in the upcoming v1.11 release.

Thanks for reporting @owenwilkins and please let us know if you run into any problems with the fix. Especially as we simply guessed what the RSEM output file looks like: https://github.com/ewels/MultiQC_TestData/pull/203/files (but that file seemed to generate the same traceback that you reported above).

Phil
